lib/experimental.ts

```diff
@@ -1,15 +1,54 @@
-import { PluginResponse } from "./types";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+import { Docker } from "./docker";
+import * as staticModule from "./static";
+import { ImageType, PluginResponse } from "./types";
 
 export async function experimentalAnalysis(
-  options: any,
+  targetImage: string,
 ): Promise<PluginResponse> {
   // assume Distroless scanning
-  return distroless(options);
+  return distroless(targetImage);
 }
 
 // experimental flow expected to be merged with the static analysis when ready
-export async function distroless(options: any): Promise<PluginResponse> {
-  // assumption #1: the image is present in the local Docker daemon
+export async function distroless(targetImage: string): Promise<PluginResponse> {
+  const archiveDir = path.join(os.tmpdir(), "snyk-image-archives");
```

Comment:
this should be validated with the customer

Reply:
what?

Reply:
that this is the place they want us to save the temporary archives.. for example - maybe the file system is readonly in the tmp folder so this will fail (in their CI/CD process)

Reply:
I'd be happy to make it configurable - but right now I'm using the standard place to store tmp files as dictated by the OS. I prefer keeping things simple and standard, once we have it working we can extend it.

Reply:
we have only one customer that is supposed to use it, I don't say we shouldn't merge it now, but I say that we need to let them know of that so we can approve that that's good for them